src/components/RocketProfile/RocketProfile.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './RocketProfile.css';
+
+function RocketProfile() {
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const bookedRockets = rockets.filter((rocket) => rocket.reserved);
```

Comment:
👍 You're on the right track with using Redux for state management! However, there's an opportunity to further enhance your code's structure:

- [ ] [Optional] Instead of handling the filter logic within your DOM component, I highly recommend moving this filtering functionality to the appropriate Redux slice. By doing so, you can centralize your state management in Redux and import the filtered data into your component. This approach adheres to the core concept of Redux, where state management is centralized, making it easier to maintain and reuse state logic across your application.

By making this adjustment, you'll have a cleaner separation of concerns and can efficiently manage state functionality within Redux. Keep up the great work! 👍🚀

Reply:
> 👍 You're on the right track with using Redux for state management! However, there's an opportunity to further enhance your code's structure:
> 
> * [ ]  [Optional] Instead of handling the filter logic within your DOM component, I highly recommend moving this filtering functionality to the appropriate Redux slice. By doing so, you can centralize your state management in Redux and import the filtered data into your component. This approach adheres to the core concept of Redux, where state management is centralized, making it easier to maintain and reuse state logic across your application.
> 
> By making this adjustment, you'll have a cleaner separation of concerns and can efficiently manage state functionality within Redux. Keep up the great work! 👍🚀

[OPTIONAL] I would like you to take note of this advice from the previous reviewer. It is important to allow redux to take full control of what it is meant to do in our applications and not to defeat its purpose. Kindly give a serious consideration to this advice. He has said it all and it is very important to stick to this advice as is going to help a lot later on. :pray: